main.go

```diff
@@ -83,6 +83,8 @@ func main() {
 		writeTerminals(gSymbols, cfg)
 	}
 
+	checkEmptyKeyword(g)
```

Comment:
Maybe this can be placed with the consistency check and return an error there?

https://github.com/goccmack/gocc/commit/101acee99d75357ecdc5acd209ee2cec8a638cc9

Then we have all the ast checks on one place